common/scala/src/main/scala/org/apache/openwhisk/core/database/ArtifactStoreProvider.scala

```diff
@@ -48,4 +48,15 @@ trait ArtifactStoreProvider extends Spi {
       None
     }
   }
+
+  protected def getLegacyAttachmentStore[D <: DocumentSerializer: ClassTag]()(
+    implicit actorSystem: ActorSystem,
+    logging: Logging,
+    materializer: ActorMaterializer): Option[AttachmentStore] = {
+    if (ConfigFactory.load().hasPath("whisk.spi.LegacyAttachmentStoreProvider")) {
```

Comment:
Added a new configuration item to load a secondary `AttachmentStoreProvider`. 

The class defined as the legacy store allows only reads. However for all new large action payloads, writes will continue to use the pre-existing `AttachmentStoreProvider` class configured via `whisk.spi.LegacyAttachmentStoreProvider`